src/test/java/org/eclipse/yasson/internal/serializer/ObjectDeserializerTest.java

```diff
@@ -0,0 +1,39 @@
+package org.eclipse.yasson.internal.serializer;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import javax.json.bind.Jsonb;
+import javax.json.bind.JsonbBuilder;
+
+public class ObjectDeserializerTest {
+
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    @Test
+    public void testGetInstanceExceptionShouldContainClassNameOnMissingConstructor() {
+        expectedException.expect(RuntimeException.class);
+        expectedException.expectMessage("org.eclipse.yasson.internal.serializer.ObjectDeserializerTest$DummyDeserializationClass");
```

Comment:
lets change this to `DummyDeserializationClass.class.getCanonicalName()` in case we ever rename/move this class in the future

Reply:
updated it to `DummyDeserializationClass.class.getName()` as the CanonicalName contains a `.` instead of a `$` and the test would fail.